lgc/patch/NggPrimShader.cpp

```diff
@@ -2885,7 +2885,7 @@ void NggPrimShader::splitEs(Module *module) {
     valueMap[&arg] = newArg++;
 
   SmallVector<ReturnInst *, 8> retInsts;
-  CloneFunctionInto(esCullDataFetchFunc, esEntryPoint, valueMap, false, retInsts);
+  CloneFunctionInto(esCullDataFetchFunc, esEntryPoint, valueMap, CloneFunctionChangeType::LocalChangesOnly, retInsts);
```

Comment:
As Carl said elsewhere, please guard this with the applicable LLVM revision number, like in #1168. Then it does not need to be synchronized with an update to the LLVM it is built with.